index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+  
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header>
+      <nav>
+        <h3 class="logo">My Logo</h3>
+        <i class="hamburguer fa fa-bars"></i>
+      </nav>
+    </header>
+    <main>
+      <section class="header-banner">
+        <h2 class="name">I'm Oscar<br>Glad to see<br>you!</h2>
+        <h1 class="introduction">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</h1>
+
+        <h3 class="connect-text">LET'S CONNECT</h3>
+        <div class="social-media-icons">
+          <a href="https://linkedin.com/in/OscarFM-dev">
+            <img src="./images/linkedin.png" alt="linkedin logo" class="linkedin-logo">
+          </a>
+          <a href="https://github.com/OscarFM-dev">
+            <img src="./images/github.png" alt="github logo">
+          </a>
+          <a href="https://twitter.com/OscarFMdev">
+            <img src="./images/twitter.png" alt="twitter logo">
+          </a>
+          <a href="https://www.instagram.com/oscarfmdev/">
+            <img src="./images/peace.png" alt="peace logo">
+          </a>
+        </div>
```

Comment:
- [ ] [REQUIRED] The list of icons should be using a `ul` tag to group them, and `li` per icon; each icon should have a link (`a` tag)!